jobs/client.go

```diff
@@ -0,0 +1,88 @@
+package jobs
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"log"
+	"net/http"
+	urlUtils "net/url"
+	"os"
+	"strconv"
+)
+
+const (
+	parserError        = "could not parse error"
+	unauthorizedAccess = "unauthorized access"
+	urlParseError      = "could not parse base url"
+)
+
+type ZipClient struct {
+	BaseUrl string
+	ApiKey  string
+}
+
+func NewZipClient() Client {
+	apiKey := os.Getenv("ZIP_RECRUITER_KEY")
+	if len(apiKey) == 0 {
+		panic("ZIP_RECRUITER_KEY env variable not set")
+	}
+	baseUrl := os.Getenv("ZIP_RECRUITER_BASE_URL")
+	if len(baseUrl) == 0 {
+		panic("ZIP_RECRUITER_BASE_URL env variable not set")
```

Comment:
It is not the best idea to add these as env variables, maybe is better to receive it on the constructor, and the user of the library load it on their envs

Reply:
Done